sig-architecture/production-readiness.md

```diff
@@ -28,6 +27,19 @@ aspects of the process.
     happens if it is subsequently upgraded again?
   - Are there tests for this?
 * Scalability
+  - Does enabling/using the feautre will result in new API calls?
```

Comment:
- We mean calls to the api-server right?
- do we want to know which components will be making the new calls?
- typo "feautre" -> "feature"